src/main/resources/provision/main.tf

```diff
@@ -0,0 +1,74 @@
+provider "aws" {
+    region = "us-east-1"
```

Comment:
This file contains too many fixed values. 
All of that has to be configurable. And moreover can be taken from `~/.aws/config` if specified